api/apis/integration/role_test.go

```diff
@@ -21,11 +21,13 @@ var _ = Describe("Role", func() {
 	)
 
 	BeforeEach(func() {
+		// TODO: This creates an arbitrary mapping that is not representative of production mapping
```

Comment:
I assume the mappings are tested for real by the e2e tests when we do things like this:
https://github.com/cloudfoundry/cf-k8s-controllers/blob/a1dd31293d3c0cd59bd3cd792f058ac693c95cd4/api/tests/e2e/apps_test.go#L43-L44

Regarding the "TODO" are we actually planning on changing this here or should we just make this a "NOTE" or something.

Reply:
The real mappings are exercised in production code, which the e2e tests hit.

Regarding the "TODO" - I'm not sure what to put here, but wanted to highlight that the map keys needed to exist, but their values are irrelevant in the integration tests.